### PR TITLE
Added UI references for testing config changes and VS Code for editin…

### PR DESCRIPTION
**Pull request by uSlackr**
…g config files

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @uSlackr, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16549).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_docs/configuration.markdown [hunk 1]
```diff
@@ -24,7 +24,7 @@ If you run into trouble while configuring Home Assistant, have a look at the [co
 
 <div class='note tip'>
 
-  Test any changes to your configuration files from the command line with `hass --script check_config`. This script allows you to test changes without the need to restart Home Assistant. Remember to run this as the user you run Home Assistant as.
+  Test any changes to your configuration files from the command line with `hass --script check_config`. This script allows you to test changes without the need to restart Home Assistant. Remember to run this as the user you run Home Assistant as.  Config changes can also be tested using the UI by navigating to Configuration, Server Controls and clicking "Check Configuration"
```
Comment (frenck) [inline source/_docs/configuration.markdown]:
```suggestion
  Test any changes to your configuration files from the command line with `hass --script check_config`. This script allows you to test changes without the need to restart Home Assistant. Remember to run this as the user you run Home Assistant as. Configuration changes can also be tested using the UI by navigating to Configuration, Server Controls and clicking "Check Configuration".
```

File: source/_docs/configuration.markdown [hunk 2]
```diff
@@ -34,14 +34,16 @@ There are many ways you can edit `configuration.yaml`. Here are three options to
 
 The simplest is to use the "File Editor" add-on. This will allow you to edit your configuration from within Home Assistant itself.
 
-You can also use Samba (you may need to install the "Samba" add-on) and your favorite file editor.
+Perhaps the most robust option is to load the Visual Studio Code add-on. VS Code offers live syntax checking and auto fill of various HA entities.
```
Comment (frenck) [inline source/_docs/configuration.markdown]:
```suggestion
Perhaps the most robust option is to load the Visual Studio Code add-on. VS Code offers live syntax checking and auto-fill of various Home Assistant entities. 
```

File: source/_docs/configuration.markdown [hunk 3]
```diff
@@ -34,14 +34,16 @@ There are many ways you can edit `configuration.yaml`. Here are three options to
 
 The simplest is to use the "File Editor" add-on. This will allow you to edit your configuration from within Home Assistant itself.
 
-You can also use Samba (you may need to install the "Samba" add-on) and your favorite file editor.
+Perhaps the most robust option is to load the Visual Studio Code add-on. VS Code offers live syntax checking and auto fill of various HA entities. 
+
+Yuou can use Samba file share (you need to install the "Samba" add-on) and your favorite file editor.
```
Comment (frenck) [inline source/_docs/configuration.markdown]:
```suggestion
You can use Samba file share (you need to install the "Samba" add-on) and your favorite file editor.
```

File: source/_docs/configuration.markdown [hunk 4]
```diff
@@ -34,14 +34,16 @@ There are many ways you can edit `configuration.yaml`. Here are three options to
...
+You can load changes to the following components without restarting busing the UI by navigating to Configuration, Server Controls and scrolling down to the YAML configuration reloading: [automations](/docs/automation/), [core (customize)](/docs/configuration/customizing-devices/), [groups](/integrations/group/), [history stats](/integrations/history_stats/), [HomeKit](/integrations/homekit/), [input_booleans](/integrations/input_boolean/), [input_datetimes](/integrations/input_datetime/), [input_numbers](/integrations/input_number/), [input_selects](/integrations/input_select/), [input_texts](/integrations/input_text/), [MQTT](/integrations/mqtt/), [persons](/integrations/person/), [scenes](/integrations/scene/), [scripts](/integrations/script/), [statistics](/integrations/statistics/), [template sensors](/integrations/template/), [timers](/integrations/timer/), [zones](/integrations/zone/), and more without restarting.
```
Comment (frenck) [inline source/_docs/configuration.markdown]:
```suggestion
You can load changes to the following components without restarting, by using the UI. Navigate to Configuration, Server Controls and scrolling down to the YAML configuration reloading: [automations](/docs/automation/), [core (customize)](/docs/configuration/customizing-devices/), [groups](/integrations/group/), [history stats](/integrations/history_stats/), [HomeKit](/integrations/homekit/), [input_booleans](/integrations/input_boolean/), [input_datetimes](/integrations/input_datetime/), [input_numbers](/integrations/input_number/), [input_selects](/integrations/input_select/), [input_texts](/integrations/input_text/), [MQTT](/integrations/mqtt/), [persons](/integrations/person/), [scenes](/integrations/scene/), [scripts](/integrations/script/), [statistics](/integrations/statistics/), [template sensors](/integrations/template/), [timers](/integrations/timer/), [zones](/integrations/zone/), and more without restarting.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/configuration.markdown (hunk 4 above)
Comment (uSlackr) [reply source/_docs/configuration.markdown]: I need to be more careful! Thanks

**Comment by uSlackr** [comment]
@frenck I'm a gihub n00b - not sure I'm doing this right.  I resolved the conversations.  Do I have to do something else?

**Comment by frenck** [comment]
Committing them, or manually editing the PR to apply the changes manually.
I'll apply them 😉

**Review by frenck — APPROVED** [review]
Thanks, @uSlackr 👍

**Pull request merged**